tmail-backend/saas/saas-rabbitmq/src/test/java/com/linagora/tmail/saas/rabbitmq/subscription/SaaSDomainSubscriptionConsumerTest.java

```diff
@@ -634,8 +758,17 @@ void verifyChainSaasDomainSubscriptionMessage() {
     private void publishAmqpSaaSDomainSubscriptionMessage(String message) {
         rabbitMQExtension.getSender()
             .send(Mono.just(new OutboundMessage(
-                EXCHANGE_NAME,
-                DOMAIN_ROUTING_KEY,
+                SaaSSubscriptionRabbitMQConfiguration.DEFAULT.exchange(),
+                SaaSSubscriptionRabbitMQConfiguration.DEFAULT.domainRoutingKey(),
+                message.getBytes(UTF_8))))
+            .block();
+    }
+
+    private void publishAmqpSaaSConfigurationMessage(String message) {
+        rabbitMQExtension.getSender()
+            .send(Mono.just(new OutboundMessage(
+                SaaSSubscriptionRabbitMQConfiguration.DEFAULT.configurationExchange(),
+                SaaSSubscriptionRabbitMQConfiguration.DEFAULT.domainConfigurationRoutingKey(),
                 message.getBytes(UTF_8))))
             .block();
     }
```

Comment:
I think the tests do not make sense anymore with the updated version of the spec?

We seem to have two distinct payloads now: 

One for domain subscription features and one for domain subscript validation status.

After technically it isn't wrong either regarding the code. I'm ok to accept it as is but just raising if it's ok or if we should be more rigorous here?